op_builder/builder.py

```diff
@@ -188,7 +187,6 @@ def installed_rocm_version():
         ROCM_MAJOR = '0'
         ROCM_MINOR = '0'
         if OpBuilder.is_rocm_pytorch():
-            from torch.utils.cpp_extension import ROCM_HOME
```

Comment:
I have a PR that uses this import (#2093). Could we preserve this? or would that cause the precommit to fail?

Reply:
I noticed it, but it would cause precommit to fail. If you get that PR merged, I can rebase and keep it without the failure.

Reply:
Thank you! Please let me know when it's merged so I can rebase and push the adjusted version of this PR.

Reply:
I've rebased against your now-merged PR. See <https://github.com/microsoft/DeepSpeed/pull/2051#issuecomment-1189682379> for more information.